charts/healthcheck/README.md

```diff
@@ -0,0 +1,43 @@
+# Health Check
+
+HealthCheck is a tool that can be used to monitor the basic health of the Service Catalog deployment.  It utilizes the User Provided Service Broker to perform basic end to end tests such as creating a Service Instance and Binding and verifying the operations are successful and then tearing it down.  It collects Prometheus metrics which can be scraped for analysis and alerting (ie error rate, execution time to create an instance or binding, etc).
+
+For more information,
+[visit the Service Catalog project on github](https://github.com/kubernetes-incubator/service-catalog).
+
+## Installing the Chart
+
+To install the chart with the release name `healthcheck`:
+
+```bash
+$ helm install charts/healthcheck --name healthcheck --namespace healthcheck --set imagePullPolicy=Never --set image=healthcheck:canary
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `healthcheck` deployment:
+
+```bash
+$ helm delete healthcheck
```

Comment:
You will need a `--purge` flag so that the release is deleted. Otherwise if they try to install it again, it will fail because the name is taken.